vault/mount.go

```diff
@@ -509,6 +509,11 @@ func (entry *MountEntry) Deserialize() map[string]interface{} {
 	}
 }
 
+// DecodeMountTable is used for testing
+func (c *Core) DecodeMountTable(raw []byte) (*MountTable, error) {
+	return c.decodeMountTable(context.Background(), raw)
```

Comment:
Sorry for constantly bringing up contexts, but should we pass a context through instead, since the internal function needs it? We can leave it up to the caller (test) to send in `context.Background()`.

Reply:
That is a good point! Will do!

Reply:
Fixed in https://github.com/hashicorp/vault/pull/21950/commits/59378ba35a638deaddf8a3f612047a92b95db144